test.py

```diff
@@ -31,31 +32,34 @@
         if transpile_output.returncode != 0:
             print(f"**** ****TEST FAILED**** ****: {file}")
             print("stdout:\n", transpile_output.stdout.decode("utf-8"), sep='')
-            print("stderr:\n", transpile_output.stderr.decode("utf-8"), sep='')
-            continue
+            print("stderr:\n", transpile_output.stderr.decode("utf-8"), sep='')         
```

Comment:
There's white space remove it.
```suggestion
            print("stderr:\n", transpile_output.stderr.decode("utf-8"), sep='')
```